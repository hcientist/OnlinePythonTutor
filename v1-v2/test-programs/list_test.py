x = [1, 2, "hello", (3, 4)]
