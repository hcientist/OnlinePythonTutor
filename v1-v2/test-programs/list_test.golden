[ { "event" : "step_line",
    "func_name" : "<module>",
    "globals" : {  },
    "line" : 1,
    "stack_locals" : [  ],
    "stdout" : ""
  },
  { "event" : "return",
    "func_name" : "<module>",
    "globals" : { "x" : [ "LIST",
            99999,
            1,
            2,
            "hello",
            [ "TUPLE",
              99999,
              3,
              4
            ]
          ] },
    "line" : 1,
    "stack_locals" : [  ],
    "stdout" : ""
  }
]
